hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/handlers/ContractGetBytecodeHandler.java

```diff
@@ -57,13 +68,36 @@ public Response createEmptyResponse(@NonNull final ResponseHeader header) {
     @Override
     public void validate(@NonNull final QueryContext context) throws PreCheckException {
         requireNonNull(context);
-        throw new UnsupportedOperationException("Not implemented");
+        validateFalsePreCheck(contractFrom(context) == null, INVALID_CONTRACT_ID);
     }
 
     @Override
     public Response findResponse(@NonNull final QueryContext context, @NonNull final ResponseHeader header) {
         requireNonNull(context);
         requireNonNull(header);
-        throw new UnsupportedOperationException("Not implemented");
+        final var contractGetBytecode = ContractGetBytecodeResponse.newBuilder().header(header);
+        if (header.nodeTransactionPrecheckCode() == OK && header.responseType() == ANSWER_ONLY) {
```

Comment:
nit: comments here would have been valuable.
Does this imply only for the answer only case are the actual bytes returned? In all other cases just the response header is sent?

Q: Where does this show up in practice?

Reply:
@Nana-EC I just followed the example of `ContractGetInfoHandler` (it's logic) in order to implement `ContractGetBytecodeHandler`, so `ContractGetBytecodeSuite` and a couple of other suites would pass.

Reply:
> Does this imply only for the answer only case are the actual bytes returned? In all other cases just the response header is sent?

This is correct @Nana-EC; although unfortunately the [`ResponseType` enum](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/query_header.proto#L39) includes an unsupported field `ANSWER_STATE_PROOF`, the "answer" to the query is returned **ONLY** when both,
  1. The requested response type is `ANSWER_ONLY`; and,
  2. The [`ResponseHeader#nodeTransactionPrecheckCode`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/response_header.proto#L41) is `OK`.

Reply:
@thenswan I'm fine with this. 
Assuming there's a follow up PR you can add such a comment to the file later.
Thanks